File: Python/Sort/Insert_sort.py
lst = [1, 5, 3, 2, 8, 9, 6, 4, 7]
print(lst)

for i in range(0, len(lst)):
    for j in range(i, 0, -1):
        if lst[j] < lst[j - 1]:
            lst[j], lst[j - 1] = lst[j - 1], lst[j]
    print(lst)
print(lst)

File: JAVA/Sort/Insert_sort.java
package JAVA.Sort;

import java.util.Arrays;

public class Insert_sort {
    public static void main(String[] args) {
        int[] lst = {1, 5, 2, 8, 6, 3, 9, 4, 7};
        System.out.println(Arrays.toString(lst));
        for (int i = 0; i < lst.length; i++) {
            for (int j = i; j > 0; j--) {
                if (lst[j] < lst[j - 1]) {
                    int tmp = lst[j];
                    lst[j] = lst[j - 1];
                    lst[j - 1] = tmp;
                }
            }
        }
        System.out.println(Arrays.toString(lst));
    }
}
